.github/workflows/build-deploy.yaml

```diff
@@ -56,3 +50,39 @@ jobs:
             ghcr.io/python-discord/site:${{ steps.sha_tag.outputs.tag }}
```

Comment:
The step was removed, so this seems like it will no longer work. Did you mean `inputs.sha_tag`? Same thing on line 82, and perhaps elsewhere that I missed.

Reply:
Absolutely right, done in 36c9988028accc5eefaa61a630ccbb355fdd0067